gobrew.go

```diff
@@ -463,10 +463,6 @@ func (gb *GoBrew) Use(version string) {
 // Version of GoBrew
 func (gb *GoBrew) Version(currentVersion string) {
 	utils.Infoln("[INFO] gobrew version is " + currentVersion)
-	if "v"+currentVersion != gb.getLatestVersion() {
```

Comment:
Skipping this because it makes an API call and response can be slow.
We already use getLatestVersion at the time of `upgrade` therefore removing it from here.

Reply:
👍